### ISSUE TITLE
Sonar collector is not working with Sonarqube 6.7

### ISSUE THREAD

**Issue by Piaoyzma** [capitalone/Hygieia#2351]
Hi,I have some problem when I used Sonar collector。I cannot get any data，except lines of code and VERSION.
![image](https://user-images.githubusercontent.com/29573171/42621615-dec17ae8-85f0-11e8-9a57-d5fc6eec4a6d.png)

I tried a lot of ways, but I didn't solve it. Please help, thank you very much.

1.I downloaded the latest code from the master. 
2.I am using sonar of version 6.7.3；
3.The configuration file is as follows；

# Database Name
dbname=dashboard

# Database HostName - default is localhost
dbhost=127.0.0.1

# Database Port - default is 27017
dbport=27017

# MongoDB replicaset
#dbreplicaset=[false if you are not using MongoDB replicaset]
#dbhostport=[host1:port1,host2:port2,host3:port3]

# Database Username - default is blank
dbusername=db

# Database Password - default is blank
dbpassword=dbpass

# Collector schedule (required)
sonar.cron=0 0/1 * * * *

# Sonar server(s) (required) - Can provide multiple
sonar.servers=http://192.168.xx.xx:9000

#sonar.host=192.168.xx.xx
#sonar.port=9000
#sonar.protocol=http

# Sonar version, match array index to the server. If not set, will default to version prior to 6.3.
sonar.versions=6.7

# Sonar Metrics - Required. 
#Sonar versions lesser than 6.3
#sonar.metrics[0]=ncloc,line_coverage,violations,critical_violations,major_violations,blocker_violations,violations_density,sqale_index,test_success_density,test_failures,test_errors,tests

# For Sonar version 6.3 and above
sonar.metrics=ncloc,violations,new_vulnerabilities,critical_violations,major_violations,blocker_violations,tests,test_success_density,test_errors,test_failures,coverage,line_coverage,sqale_index,alert_status,quality_gate_details

# Sonar login credentials
sonar.username=admin
sonar.password=admin

**Comment by Sbrenthughes** [comment]
Do you see any errors in the sonar log?

**Comment by Piaoyzma** [comment]
@Sbrenthughes thanks,I followed your suggestion and tried it, but it didn't seem to work. The sonar log is as follows, just as I did before the replacement.

![image](https://user-images.githubusercontent.com/29573171/42668763-f3b6a9fc-8684-11e8-9571-4c1e22fb9130.png)

![image](https://user-images.githubusercontent.com/29573171/42668758-e6692dec-8684-11e8-8679-de1ccd7e33a1.png)

**Comment by Piaoyzma** [comment]
@Sbrenthughes hi!After I deleted the configuration of the UI, I re-selected it and then suddenly it was fine. It's really amazing. I suspect that there may be some problems with caching. I changed the git repository address when collecting git data, but the data did not become a new repository. Thanks again^^
![image](https://user-images.githubusercontent.com/29573171/42668939-c2813bda-8685-11e8-8b9d-0f576ef0bf3c.png)
